walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -27,17 +27,20 @@
 import java.util.stream.Collectors;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+  public Collection<TimeRange> findTimeRangesForMeeting(Collection<Event> events, MeetingRequest request) {
     EventsTimeRange eventsTimeRange = filterEventsTime(events, request.getAttendees(), request.getOptionalAttendees());
 
     Collection<TimeRange> times = findTimes(eventsTimeRange.getAllEventsTimeRange(), request.getDuration());
     if (times.isEmpty()) {
-      System.out.println(eventsTimeRange.getMandatoryEventsTimeRange());
       times = findTimes(eventsTimeRange.getMandatoryEventsTimeRange(), request.getDuration());
     }
     return times;
   }
-  
+
+  /**
+   * Filter events which attand at list one of attendees. Collect event to mandatoryEventsTimeRange if mandatory attendee has this meeting event, 
+   * to optionalEventsTimeRange if optional attendee has this meetang. MandatoryEventsTimeRange for TimeRanges while attendees are busy, optionalEventsTimeRange for TimeRanges while optional attendees are busy.
```

Comment:
Please limit the line length so you don't have to scroll horizontally. The style guide recommends 100 columns: https://google.github.io/styleguide/javaguide.html#s4.4-column-limit